src/openrct2/ride/Vehicle.cpp

```diff
@@ -6393,26 +6396,26 @@ int32_t Vehicle::UpdateMotionDodgems()
  *
  *  rct2: 0x006DD365
  */
-bool vehicle_update_dodgems_collision(Vehicle* vehicle, int16_t x, int16_t y, uint16_t* spriteId)
+bool Vehicle::UpdateDodgemsCollision(int16_t curX, int16_t curY, uint16_t* spriteId)
```

Comment:
another refactoring project is replacing these x and y variables with the respective structs. In this instance this is a `CoordsXY`. Can you replace x and y with a `CoordsXY`. All calls can construct this with `{x ,y}`

Reply:
During this I actually noticed this does not Update the vehicle in any way, so I renamed it, and added a const to it.